### ISSUE TITLE
Ready to use dashboards for metrics components in SigNoz

### ISSUE THREAD

**Issue by srikanthccv** [SigNoz/dashboards#1]
Our collector distribution enables receiving metrics from different system components. Once the data is received, there is no easy way to visualise the data quickly. One has to manually create all the panels and make it work. This is usually the same for almost everyone. We can improve the experience for users by giving them ready-to-use dashboards for all components as we do for the host metrics. 

@pranay01 @prashant-shahi

**Comment by pranay01** [comment]
Good idea! Will metrics provided by otel collector work for the metrics which default grafana dashboards for those db/apps plot? Might be a good starting point

For example, if couchdbreceiver is enabled in otel collector, will this dashboard work? - https://grafana.com/grafana/dashboards/10768-couchdb-dashboard/
